commons-geometry-euclidean/src/test/java/org/apache/commons/geometry/euclidean/threed/Vector3DTest.java

```diff
@@ -1146,6 +1145,65 @@ void testEqualsAndHashCode_signedZeroConsistency() {
         Assertions.assertEquals(b.hashCode(), d.hashCode());
     }
 
+    @Test
+    void testHashCodeCollisions_symmetricAboutZero() {
+        final double any = Math.random();
```

Comment:
Note that this could create sporadic failures. We can formalise this using some random numbers:
```java
@ParameterizedTest
@ValueSource(doubles = {1.23, 4.56, 42, Math.PI})
void testHashCodeCollisions_symmetricAboutZero(double any) {
```